src/components/DuosHeader.js

```diff
@@ -75,6 +191,19 @@
       isDataOwner = currentUser.isDataOwner;
     }
 
+    const dropdownLinks = {
+      statistics: {
+        'Votes Statistics': {
+          isRendered: true,
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/DataBiosphere/duos-ui/pullRequest?prid=6963768)
```suggestion
          link: "/summary_votes"
```

Reply:
1) Suggestion is not correct. 2) Single quotes as strings are accepted for most, if not all JS linters as accepted syntax. Most libraries use single quotes over double-quotes (https://bytearcher.com/articles/single-or-double-quotes-strings-javascript). It's going to be an issue if JS devs are onboarded, write up code that is accepted as standard, only to get a slew of errors that don't reflect the norm

Reply:
These string suggestion comments are not actually me ... they are codacy posting as me. I agree ... will look at fixing codacy so it doesn't try do what eslint is warning against.

Reply:
I know, just wanted to point out the error and list references for why devs use single-quotes.

Reply:
Still working on codacy ... it's not re-analyzing correctly even after disabling the rulesets.